src/engraving/libmscore/scoreorder.cpp

```diff
@@ -0,0 +1,437 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include <iostream>
+
+#include "scoreorder.h"
+
+#include "libmscore/score.h"
+#include "libmscore/part.h"
+#include "libmscore/staff.h"
+#include "libmscore/bracketItem.h"
+#include "libmscore/instrtemplate.h"
+#include "libmscore/undo.h"
+#include "libmscore/xml.h"
+
+namespace Ms {
+static const QString SOLOISTS_ID("<soloists>");
+static const QString UNSORTED_ID("<unsorted>");
+
+//---------------------------------------------------------
+//   readBoolAttribute
+//---------------------------------------------------------
+
+bool ScoreOrder::readBoolAttribute(Ms::XmlReader& reader, const char* name, bool defvalue)
+{
+    if (!reader.hasAttribute(name)) {
+        return defvalue;
+    }
+    QString attr { reader.attribute(name) };
+    if (attr.toLower() == "false") {
+        return false;
+    } else if (attr.toLower() == "true") {
+        return true;
+    } else {
```

Comment:
You can remove this else:

} else if (attr.toLower() == "true") {
...
}

qDebug("invalid value \"%s\" for attribute \"%s\", using default \"%d\"", qPrintable(attr), qPrintable(name), defvalue);
return defvalue;
}

Reply:
Done